src/pages/Home/components/Filter/Filter.tsx

```diff
@@ -85,6 +85,7 @@ const Filter = (props: IFilterProps) => {
           value: id,
           label: mappedSupplies[id]?.name,
         })),
+        showDisabled: data.showDisabled ?? true,
```

Comment:
Esse null coalescing só faz sentido se `showDisabled` puder ser `null` ou `undefined`. Caso não, não seria o caso de apenas repassar o valor adiante?
```suggestion
        showDisabled: data.showDisabled,
```
Ou poderia só usar `!!` para forçar ser um boolean independente do valor, que contempla também outros falsy values como `''` e `0` (não sei como vem do backend e como está o type).

Reply:
Bem observado @lucianomlima! A princípio botei o nullish coalescing justamente para caso algum valor undefined chegue a essa trecho, mas acabou sendo apenas um preciosismo meu. Irei alterar